tools/bugzilla-reports-tool/.gitignore

```diff
@@ -0,0 +1,105 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Could we remove this file? As the root `.gitignore` would handle most cases. If they are handled, please add it to the `.gitignore` that is present at root level.